src/executor.rs

```diff
@@ -453,8 +459,10 @@ where
         if let Err(ret) = self.uc.remove_hook(self.cmp_hook) {
             warn!("Fail to uninstall cmp tcg opcode hook due to {ret:?}");
         }
-        if let Err(ret) = self.uc.remove_hook(self.new_tb_hook) {
-            warn!("Fail to uninstall edge gen hook due to {ret:?}");
+        if let Some(new_tb_hook) = self.new_tb_hook.take() {
+            if let Err(ret) = self.uc.remove_hook(new_tb_hook) {
+                warn!("Fail to uninstall edge gen hook due to {ret:?}");
```

Comment:
we have implemented `uc_strerror` for `err` on dev. maybe we can remove `:?` for all such logs?

Reply:
This is done in the cmplog PR which has not been merged. Anyway, I have also fixed it in this PR.